frontend/src/styles/app.scss

```diff
@@ -70,7 +70,49 @@ h5 {
   }
 }
 
-a {
+// reset default button style
+button {
```

Comment:
@pregnantboy would you rather that the `a` links be converted into a button component? Seems like the style you were going for.